tests/data/tagged_codelist/tag_species.yaml

```diff
@@ -0,0 +1,7 @@
+- Species:
+  - CO2:
+     definition: carbon dioxide (CO2)
+     unit: Mt CO2
```

Comment:
```suggestion
        definition: carbon dioxide (CO2)
        unit: Mt CO2/yr
```

Reply:
Thanks. There was also inconsistent indentation in tag_species.yaml that was causing errors. I had already made fixes locally. I have committed and pushed those now, so I haven't committed your suggestions.

My commit is slightly different from yours. I kept `unit: {Species}/yr` in the `Emissions|{Species}` tagged variable, and instead removed `/yr` from both unit attributes in tag_species.yaml. The species tag could in principle also be used for variables representing cumulative emissions, emission intensity, etc. Those could all have units with `Mt {Species}` in the denominator, but would need different or no units in the denominator.

It doesn't matter here of course, so I'm happy to move where `/yr` goes if you have a strong preference.

Reply:
Good point, thanks.